### DISCUSSION TITLE
稍后再看

### DISCUSSION THREAD

**Discussion by bigbug-dev** [the1812/Bilibili-Evolved#1640] [综合讨论 / General]
如图，由稍后再看进入的视频界面插件不完整？与平时的界面不一样，偏向原版：
![image](https://user-images.githubusercontent.com/62877853/110940505-faf78280-8371-11eb-9ead-103864657142.png)
![image](https://user-images.githubusercontent.com/62877853/110940534-064aae00-8372-11eb-9a3f-040fe336af13.png)
并且某些视频无法由稍后再看观看：
![image](https://user-images.githubusercontent.com/62877853/110940646-33975c00-8372-11eb-9193-f5d8ab9f08f1.png)
![image](https://user-images.githubusercontent.com/62877853/110940674-3bef9700-8372-11eb-8ade-63f979195b4d.png)

**Comment by the1812** [comment]
那是旧版稍后再看 我忘记换链接了 (怎么现在才有人发现

**Reply by bigbug-dev** [reply]
说明稍后再看这个功能真的没人用哈哈

**Reply by the1812** [reply]
我猜大部分人都开着稍后再看重定向就没发现

**Reply by bigbug-dev** [reply]
刚刚打开重定向看了一下确实没这个问题了
